### PR TITLE
Linux: fix mentions to stats coming from procfs.

### PR DESCRIPTION
**Pull request by Oppen**
Some comments and the accumulator name still reflected the previous, procfs based, drop statistics origin.

### CODE REVIEW

File: pcap-linux.c [hunk 1]
```diff
@@ -2426,8 +2426,8 @@ pcap_stats_linux(pcap_t *handle, struct pcap_stat *stats)
 	 *	"ps_drop" is not supported.
 	 *
 	 *	"ps_ifdrop" is supported. It will return the number
-	 *	of drops the interface reports in /proc/net/dev,
-	 *	if that is available.
+	 *	of drops the interface reports in /sys/class/net, if that
```
Comment (guyharris) [inline pcap-linux.c]: Perhaps that should be "...the interface reports in /sys/class/net/statistics/{if_name}/rx_{missed,fifo}_errors", as per another comment.

File: pcap-linux.c [hunk 2]
```diff
@@ -303,7 +303,7 @@ typedef int		socklen_t;
  */
 struct pcap_linux {
 	u_int	packets_read;	/* count of packets read with recvfrom() */
-	long	proc_dropped;	/* packets reported dropped by /proc/net/dev */
+	long	sysfs_dropped;	/* packets reported dropped by /sys/class/net */
```
Comment (guyharris) [inline pcap-linux.c]: Perhaps that should be "...reported dropped in /sys/class/net/statistics/{if_name}/rx_{missed,fifo}_errors", as per another comment.

File: pcap-linux.c [hunk 3]
```diff
@@ -1633,10 +1633,10 @@ pcap_activate_linux(pcap_t *handle)
 	/*
 	 * If we're in promiscuous mode, then we probably want
 	 * to see when the interface drops packets too, so get an
-	 * initial count from /proc/net/dev
+	 * initial count from /sys/class/net
```
Comment (guyharris) [inline pcap-linux.c]: Perhaps that should be "initial count from /sys/class/net/statistics/{if_name}/rx_{missed,fifo}_errors", as per another comment.

File: pcap-linux.c [hunk 4]
```diff
@@ -2331,13 +2331,13 @@ pcap_stats_linux(pcap_t *handle, struct pcap_stat *stats)
 	long if_dropped = 0;
 
 	/*
-	 *	To fill in ps_ifdrop, we parse /proc/net/dev for the number
+	 *	To fill in ps_ifdrop, we parse /sys/class/net for the number
```
Comment (guyharris) [inline pcap-linux.c]: Perhaps that should be "...we parse /sys/class/net/statistics/{if_name}/rx_{missed,fifo}_errors for the numbers", or something that separately mentions the two statistics.

**Comment by Oppen** [comment]
Fair enough. Would you prefer for me to rebase it or pile another commit up?

**Comment by guyharris** [comment]
> Fair enough. Would you prefer for me to rebase it or pile another commit up?

For comments, I'm OK either way - do what works more easily for you.

**Comment by Oppen** [comment]
How about now? I also fixed a previous comment, mentioning `statistics/{if_name}` instead of `{if_name}/statistics`.

File: pcap-linux.c [hunk 5]
```diff
@@ -2331,13 +2331,15 @@ pcap_stats_linux(pcap_t *handle, struct pcap_stat *stats)
 	long if_dropped = 0;
 
 	/*
-	 *	To fill in ps_ifdrop, we parse /proc/net/dev for the number
+	 * To fill in ps_ifdrop, we parse
+	 * /sys/class/net/{if_name}/statistics/rx_{missed,fifo}_errors
+	 * for the number
```
Comment (guyharris) [inline pcap-linux.c]: "numbers", plural.

**Comment by guyharris** [comment]
> How about now? I also fixed a previous comment, mentioning `statistics/{if_name}` instead of `{if_name}/statistics`.

Just one more plural tweak.

And then I'll go change my comments to use {if_name} for more clarity and for stylistic consistency.

**Pull request merged**